tgui/packages/tgui/interfaces/PipeDispenser.js

```diff
@@ -0,0 +1,102 @@
+import { useBackend, useLocalState } from '../backend';
```

Comment:
Could you include a license header? See other interfaces for examples, and feel free to ask if you're unsure.

Reply:
Noticed quite a few tgui files without licenses. How do licenses work here?

Reply:
mostly we just put MIT on all tg stuff, same as upstream - everything is CC-BY-NC-SA unless explicitly licensed otherwise

Reply:
ill try mit